src/testUtils/java/org/broadinstitute/hellbender/testutils/IntegrationTestSpec.java

```diff
@@ -81,6 +90,14 @@ public void setValidationStringency(final ValidationStringency validationStringe
         this.validationStringency = validationStringency;
     }
 
+    public void setWriteToGcs(boolean writeToGcs) {
+        this.writeToGcs = writeToGcs;
+    }
+
+    public boolean getWriteToGcs() {
+        return this.writeToGcs;
+    }
+
```

Comment:
These appear to be unreferenced (left over from a previous attempt to path-enable this ?)

Reply:
Good eye, thank you. I have removed them.